Mojave/README.md

```diff
@@ -13,6 +13,10 @@ Note: Use `create_iso_mojave.sh` for generating a macOS Mojave based ISO image.
 
 Note: Use 64-bit Ubuntu 18.04.2 LTS as the host for "best" results.
 
+Note: Tested working on Ubuntu 19.04 host.
+
+With either method, edit the desired Clover screen resolution in `./config.plist.stripped.qemu` (and set to match in EDK II OVMF UEFI, or else bad things happen when resolution changes on Clover startup in fullscreen).
```

Comment:
What is this 'either method' in this context?

Reply:
"either method" refers to the old or new method of generating Clover disk mentioned in the same document.

Reply:
You probably need to edit the document to make that clear.

Reply:
Thanks. Documented at https://github.com/kholia/OSX-KVM/blob/master/notes.md#higher-resolution-uefi--clover now.

Reply:
Thanks. Sorry for the silence, busy weekend. I'll revisit this to see if I can further clarify if I find the time.